### PR TITLE
Cheat sheet

### PR DESCRIPTION
**Pull request by weshayutin**
* WIP cheat sheet that customers can use while they are working with us.
* Maybe some day something for https://developers.redhat.com/cheat-sheets/
* Can be generally handy but needs additional eyes and contributions
* source: https://hackmd.io/6-i34q7uR_2um-yH1trmbA

### CODE REVIEW

File: docs/oadp_cheet_sheet.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+  ttl: 720h0m0s
+```
+Create the backup
+```
+oc create -f backup.yaml
+```
+
+#### Create a restore
+Update the following restore.yaml CR
+```
```
Comment (mateusoliveira43) [inline docs/oadp_cheet_sheet.md:38]:
```yaml for format highlight

File: docs/oadp_cheet_sheet.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+oc get dpa -n openshift-adp
+```
+Get the details:
+```
+oc get dpa dpaname -n openshift-adp -o yaml
+```
+
+#### Create a backup
+Update the following backup.yaml CR
+```
```
Comment (mateusoliveira43) [inline docs/oadp_cheet_sheet.md:18]: yaml for format highlight

**Comment by mateusoliveira43** [comment]
should we add a link in README.md in root or when users ask for something like this in slack, we point them?

File: docs/oadp_cheet_sheet.md (hunk 2 above)
Comment (weshayutin) [reply docs/oadp_cheet_sheet.md]: thanks!

File: docs/oadp_cheet_sheet.md (hunk 1 above)
Comment (weshayutin) [reply docs/oadp_cheet_sheet.md]: thanks!

File: docs/oadp_cheat_sheat.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
+[![hackmd-github-sync-badge](https://hackmd.io/6-i34q7uR_2um-yH1trmbA/badge)](https://hackmd.io/6-i34q7uR_2um-yH1trmbA)
```
Comment (kaovilai) [inline docs/oadp_cheat_sheat.md:1]: Simultaneous editing by maintainers can be achieved within GitHub PR, not sure if it's necessary to duplicate effort to hackmd and have to then PR it back here.

Comment (weshayutin) [reply docs/oadp_cheat_sheat.md:1]: true... I'll turn off the hackmd sync

File: docs/oadp_cheat_sheat.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
+## Cheat sheet
+# OADP commands
+This cheat sheet presents a list of command-line executables that are frequently used by OpenShift Administrators using OADP. The commands are organized by category.
+The OpenShift cheat sheet can be found [here](https://access.cdn.redhat.com/content/origin/files/sha256/e1/e1410185092472c9a943b85cd6b60196f3938ffa8d650026818d5456e66e01c1/openshift_cheat_sheet_r5v1.pdf?_auth_=1687811315_27797f50bfbfc2691c084156de530f76)
```
Comment (kaovilai) [inline docs/oadp_cheat_sheat.md:4]:
```suggestion

You may also find OpenShift cheat [here](https://access.cdn.redhat.com/content/origin/files/sha256/e1/e1410185092472c9a943b85cd6b60196f3938ffa8d650026818d5456e66e01c1/openshift_cheat_sheet_r5v1.pdf?_auth_=1687811315_27797f50bfbfc2691c084156de530f76) helpful.
```

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: docs/oadp_cheat_sheat.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+Velero-Native Snapshots: <none included>
+```
+
+
+
+## Data Mover Specific commands
+
+#### Clean up datamover related objects
+```
+oc delete vsb -A --all; oc delete vsr -A --all; oc delete vsc -A --all; oc delete vs -A --all; oc delete replicationsources.volsync.backube -A --all; oc delete replicationdestination.volsync.backube -A --all
```
Comment (sseago) [inline docs/oadp_cheat_sheat.md:200]: Add note that users will only want to clean up vs and vsc if they never use csi snapshots without datamover in OADP and they never create volume snapshots outside of OADP.

File: docs/oadp_cheat_sheat.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+velero backup delete $backup_name
+```
+
+## Debug a Failed or Partially Failed Backup
+
+Two simple steps should provide the information required
+
+#### Logs
+```
+velero backup logs $backup
```
[the remark this answers is not in the document]
Comment (weshayutin) [reply docs/oadp_cheat_sheat.md:132]: thanks Scott!

File: docs/oadp_cheat_sheat.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+
+```
+Name:         mssql-persistent
+Namespace:    openshift-adp
+Labels:       velero.io/storage-location=velero-sample-1
+Annotations:  velero.io/source-cluster-k8s-gitversion=v1.23.5+b0357ed
+              velero.io/source-cluster-k8s-major-version=1
+              velero.io/source-cluster-k8s-minor-version=23
+
+Phase:  [31mPartiallyFailed[0m (run `velero backup logs mssql-persistent` for more information)
```
[the remark this answers is not in the document]
Comment (weshayutin) [reply docs/oadp_cheat_sheat.md:148]: thanks Scott!

File: docs/oadp_cheat_sheat.md (hunk 5 above)
Comment (weshayutin) [reply docs/oadp_cheat_sheat.md:201]: added some warnings, explanation and alternative

File: docs/oadp_cheat_sheat.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+```
+
+#### Create a backup excluding the velero and default namespaces.
+```  
+velero backup create $backup_name --exclude-namespaces velero,default
+```
+
+#### Create a backup based on a schedule named daily-backup.
+```
+velero backup create --from-schedule $backup_name
```
[the remark this answers is not in the document]
Comment (kaovilai) [reply docs/oadp_cheat_sheat.md:108]:
```suggestion
velero backup create --from-schedule daily-backup
```

File: docs/oadp_cheat_sheat.md [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+```
+
+## Create a Backup 
+
+#### Backup with Defaults
+```
+velero backup create backup $backup_name --include-namespaces $namespace
+```
+
+#### Backup filesystem only
```
[the remark this answers is not in the document]
Comment (kaovilai) [reply docs/oadp_cheat_sheat.md:96]:
```suggestion
#### Backup using restic for PV data
```

**Review by kaovilai — APPROVED** [review]

**Pull request merged**